src/core/unit.h

```diff
@@ -365,11 +365,8 @@ typedef struct Unit {
         nsec_t cpu_usage_base;
         nsec_t cpu_usage_last; /* the most recently read value */
 
-        /* Most recently read value of memory.peak */
-        uint64_t memory_peak_last;
-
-        /* Most recently read value of memory.swap.peak */
-        uint64_t memory_swap_peak_last;
+        /* Most recently read value of memory accounting metrics */
+        uint64_t memory_accounting_last[_CGROUP_MEMORY_ACCOUNTING_METRIC_CACHED_LAST + 1];
```

Comment:
hm, why the +1?

Reply:
`_CGROUP_MEMORY_ACCOUNTING_METRIC_CACHED_LAST` is the index of the last "cached" member in enum.

Reply:
a, right